dev-tools/cmd/dashboards/export_dashboards.go

```diff
@@ -117,7 +117,7 @@ func decodeValue(data common.MapStr, key string) {
 		return
 	}
 	s := v.(string)
-	var d common.MapStr
+	var d interface{}
 	json.Unmarshal([]byte(s), &d)
```

Comment:
This code is lax on error handling. Checking the `Unmarshal` result and returning an error or panicking would be helpful.